### PR TITLE
Update SEO page for Ad Blocker (Fixes #6879)

### LINKED ISSUE #6879: Update SEO page for Ad Blocker
## Description

The copy on this page is out of date since the developments released in 65. https://www.mozilla.org/en-US/firefox/features/adblocker/ 

Copy doc:
https://docs.google.com/document/d/1ZSvg6Qh9Rbqzu_7Mqo5CNEitAH7ryxNg5DyONmnJ6EA/edit

### Estimates

| Estimates |  |
| ------------- | ------------- |
| TL | |
| PM | |
| ENG | 3 | (string extraction)
| SEO | |
| ANA | |
| DES | |
| COPY | |

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] This page is updated so the DE team can localize it with the new content
- [ ] Meta data does not change
- [ ] Strings are extracted to Pontoon and tagged as optional for localizers
- [ ] Anika on the EU team is notified so she can add strings for the DE version in Pontoon

#### Tasks
- [ ] Code it with the provide copy
- [ ] As with the 64-bit SEO page, add a download button in the hero section
- [x] Confirm w. @rraue that this SEO page can be opened to all locales in Pontoon 
- [x] Verify that meta data does not need to change

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
updates copy and adds screenshots to firefox/features/adblocker/

## Issue / Bugzilla link
#6879 

## Testing
- Demo: https://bedrock-demo-achurchwell.oregon-b.moz.works/en-US/firefox/features/adblocker/
- 🍝 copy pasta errors?
- images are compressed & high res

### CODE REVIEW

**Comment by amychurchwell** [comment]
Adding 'Do Not Merge' label until stakeholder review has finished. 
(Aleks is in process of proof-reading.)

**Comment by amychurchwell** [comment]
Approved by stakeholder.

File: bedrock/firefox/templates/firefox/features/adblocker.html [hunk 1]
```diff
@@ -24,133 +24,153 @@
...
-    <p>{{ _('Ads can slow down sites while they load, or get in the way of a streamlined browsing experience.') }}</p>
-    <p>{{ _('Ads can also be flashy and annoying, distracting you from the stuff you really want to see.') }}</p>
-    <blockquote>{{ _('“Ads can slow down sites… or get in the way of a streamlined browsing experience.”') }}</blockquote>
+    <h2 class="section-title">{{_('Choose your level of protection') }}</h2>
     <p>
-    {% trans %}
-    Finally, ad companies can track your movements online. It’s called "re-targeting" and it works like this: You visit a site and click on a product. That site puts a small piece of code in your browser called a cookie. When you surf the web, sites will know you clicked on that product and they’ll display ads for it.
-    {% endtrans %}
+      {% trans %}
+      To start, click on the Firefox menu in the top right-hand corner of your screen. It looks like four lines stacked on top of each other. In the drop-down menu, click on Content Blocking. You should see a blue pop-up with different selections.
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/features/adblocker.html]: I'm not sure I would call this a "pop up" since it actually opens a whole new page in a new tab, but maybe I'm splitting hairs.

File: bedrock/firefox/templates/firefox/features/adblocker.html [hunk 2]
```diff
@@ -24,133 +24,153 @@
...
-      Many media and news organizations make most of their money from advertising. If you block ads on their sites, they won’t get paid. If enough people block ads, business could suffer, and so will the quality of reporting and writing. Luckily, most ad blockers let you make exceptions for your favorite sites.
+      {% trans url=url('firefox.features.private-browsing') %}
+      If ads don’t bother you and you don’t mind being followed by trackers and third-party cookies, then the Standard setting should work for you. To get trackers off your tail in Standard mode, use a <a href="{{ url }}">Private Browsing</a> window.
       {% endtrans %}
     </p>
   </section>
 
   <section class="mzp-l-content l-content-image">
-    {{ high_res_img('firefox/features/privacy-illustration2.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
+    {{ high_res_img('firefox/features/adblocker/content-blocking.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/features/adblocker.html]: This screenshot is from Developer Edition, shown in the box for Strict mode. Nightly doesn't seem to mention the browser by name so maybe that's changed.

File: bedrock/firefox/templates/firefox/features/adblocker.html [hunk 3]
```diff
@@ -24,133 +24,153 @@
...
+    <p>
+      {% trans %}
+      The custom setting give you the ultimate choice. You to decide what you’re blocking and if you’re blocking trackers, cookies or both. If you allow cookies from a website, you’ll automatically be in Custom mode.
+      {% endtrans %}
+    </p>
   </section>
 
   <section class="mzp-l-content l-content-image">
-    {{ high_res_img('firefox/features/laptop.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
+    {{ high_res_img('firefox/features/adblocker/content-blocking-custom.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/features/adblocker.html]: This screenshot is from Dev Edition and shows the Cryptominer and Fingerprinter blocking features that haven't landed in the release channel yet. The copy doesn't reference those new features and just says "blocking trackers, cookies or both" indicating only two options. 

We could just update the screenshot with one from Fx 66, but I'd also suggest tweaking the copy to be more future-proof since I think these new blocking features are scheduled for Fx 67.

File: bedrock/firefox/templates/firefox/features/adblocker.html [hunk 4]
```diff
@@ -24,133 +24,153 @@
...
-    include_cta=True
-  ) %}
-    <div class="download-firefox">
-      {{ download_firefox(download_location='primary cta') }}
-    </div>
-  {% endcall %}
+  <section class="mzp-l-content l-content-narrow">
+    <h2 class="section-title">{{_('Take a bite out of cookies') }}</h2>
+    <p>
+      {% trans url='https://support.mozilla.org/en-US/kb/storage?redirectlocale=en-US&as=u&redirectslug=permission-store-data&utm_source=inproduct' %}
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/features/adblocker.html]: Omit `en-US` from the SUMO link.

File: bedrock/firefox/templates/firefox/features/adblocker.html [hunk 5]
```diff
@@ -24,133 +24,153 @@
...
-          {% trans url='https://blog.mozilla.org/firefox/ad-blocker-roundup-5-adblockers-that-improve-your-internet-experience/'%}
-          <a href="{{ url }}">We’ve compiled a list of trusted ad blocker add-ons</a>. These have been approved and tested by the Firefox community. Remember, you can get add-ons for your browser anywhere, but be extra careful when installing them. It’s not a good idea to install an untested or unapproved add-on.
-          {% endtrans %}
-        </p>
-      </div>
-    </div>
+  <section class="mzp-l-content l-content-narrow">
+    <h2 class="section-title">{{_('Send a Do Not Track signal') }}</h2>
+    <p>
+      {% trans url='https://support.mozilla.org/en-US/kb/how-do-i-turn-do-not-track-feature' %}
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/features/adblocker.html]: Omit `en-US`

File: media/css/firefox/features/adblocker.scss [hunk 6]
```diff
@@ -66,28 +57,15 @@ blockquote {
...
+    }
 }
 
-.mzp-c-billboard-image {
-    width: 100%;
-}
-
-.section-title {
-    @include text-display-lg;
+h2.section-title {
```
Comment (craigcook) [inline media/css/firefox/features/adblocker.scss]: Does this need the `h2` qualifier? Is it just for higher specificity or are there other elements with the same class and we need to distinguish them?

File: bedrock/firefox/templates/firefox/features/adblocker.html [hunk 7]
```diff
@@ -24,133 +24,153 @@
...
 
+  <section class="mzp-l-content l-content-narrow">
+      <h2 class="section-title">{{_('Speed up thanks to ad blockers') }}</h2>
+      <p>
+        {% trans %}
+        In some cases, an ad blocker can help your browser go faster. When an ad is loading, it can slow down a website. At the same time, it takes longer to find what you’re looking for if you’re too busy closing yet another ad.
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans url='https://www.mozilla.org/en-US/firefox/new/' %}
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/features/adblocker.html]: This should be `url('firefox.new')`

File: bedrock/firefox/templates/firefox/features/adblocker.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/features/adblocker.html]: Good catch!

File: bedrock/firefox/templates/firefox/features/adblocker.html [hunk 8]
```diff
@@ -24,133 +24,153 @@
...
-        <li>
-          <span>{{_('Never:') }} </span>
-          {{_('Turns off Tracking Protection for all sites.') }}
-        </li>
-      </ul>
-    </ol>
+    <h2 class="section-title">{{_('Do-it-yourself Custom mode') }}</h2>
+    <p>
+      {% trans %}
+      The custom setting give you the ultimate choice. You to decide what you’re blocking and if you’re blocking trackers, cookies or both. If you allow cookies from a website, you’ll automatically be in Custom mode.
```
[the remark this answers is not in the document]
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/features/adblocker.html]: Going with "This allows you to decide", thanks for the catch!

File: bedrock/firefox/templates/firefox/features/adblocker.html (hunk 1 above)
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/features/adblocker.html]: 🤷‍♀️ I'm not sure what would be better?

File: bedrock/firefox/templates/firefox/features/adblocker.html (hunk 3 above)
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/features/adblocker.html]: I took these screenshots in Fx 67, so I think we should keep it and maybe update the copy.
I will ping in the copy doc to include Cryptominer and Fingerprinter.

File: bedrock/firefox/templates/firefox/features/adblocker.html (hunk 2 above)
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/features/adblocker.html]: whoops! Looks like I missed one. I will retake this screenshot in Fx67.

File: bedrock/firefox/templates/firefox/features/adblocker.html (hunk 3 above)
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/features/adblocker.html]: @craigcook 
The copywriter, Aleks, brought up an excellent point:

>  I'm wondering if it would create confusion to bring cryptomining into the conversation? My instinct says people who are searching to find out what an ad blocker is won't know about crypto.

I am inclined to agree that maybe it would create confusion. Do you think this is a blocker?

File: media/css/firefox/features/adblocker.scss (hunk 6 above)
Comment (amychurchwell) [reply media/css/firefox/features/adblocker.scss]: I had needed the higher specificity when playing with the headline styles, it's no longer necessary. :+1:

File: bedrock/firefox/templates/firefox/features/adblocker.html (hunk 3 above)
Comment (craigcook) [reply bedrock/firefox/templates/firefox/features/adblocker.html]: Well the real problem is that the copy and the screenshot don't match. The copy only covers "trackers, cookies or both" which implies two options when there are four. And I do think trying to call out cryptomining or fingerprinting will further complicate it, so maybe we don't need to mention them but could at least rephrase it to be more open-ended and not imply only two things can be blocked.

Maybe something like "You can decide what you’re blocking, including trackers, cookies and more." ?

Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/features/adblocker.html]: Aleks agreed with your suggested-- updated to "You can decide what you’re blocking, including trackers, cookies and more."

File: bedrock/firefox/templates/firefox/features/adblocker.html (hunk 1 above)
Comment (craigcook) [reply bedrock/firefox/templates/firefox/features/adblocker.html]: "panel"?

Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/features/adblocker.html]: I don't really think it's a panel either. Maybe we should ask Aleks? Personally, I think describing it as a new tab would work.
